src/components/PupilComponents/QuizResultMCQ/QuizResultMCQ.test.tsx

```diff
@@ -15,6 +15,7 @@ describe("QuizResultMCQ", () => {
         <QuizResultMCQ
           answers={mcqTextAnswers}
           feedback={[null, null, null, null]}
+          pupilAnswer={[1]}
```

Comment:
Same issue here . This should probably have feedback ["incorrect", "incorrect","incorrect","incorrect"]